Rakefile

```diff
@@ -45,11 +45,11 @@ namespace :ext do
 
   task :have_jdk6_or_higher? do
     log_ok("Checking that at least java 6 is installed") do
-      version_match = `java -version 2>&1`.match(/java version "1\.(\d)\.(\d+_\d+)"/)
+      version_match = `java -version 2>&1`.match(/(java|openjdk) version "1\.(\d)\.(\d+_\d+)"/)
```

Comment:
@bradurani which openjdk version are you on? When I test with 1.6 I get `java version "1.6.0_40"`